README.rst

```diff
@@ -64,7 +64,7 @@ On a fresh Ubuntu install, you will need to install these:
 
 .. code-block:: bash
 
-	$ apt install python3 python3-pip python3-venv build-essential
+	$ apt install python3.7 python3-pip python3.7-venv build-essential scons
```

Comment:
There is no need to install scons here given we install the right version through the `pip install -r requirements.txt` we do from within the virtual env